File: app/src/main/java/com/wyl/android/MainActivity.java
package com.wyl.android;

import android.os.Bundle;
import android.os.PersistableBundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

/**
 * 项目名称：android-learn
 * 创建人：江心才子
 * 创建时间：2019-09-19 13:32
 * 内容描述：
 * 修改说明：
 */
public class MainActivity extends AppCompatActivity {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    protected void onStart() {
        super.onStart();
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
    }


}

File: example_word/src/main/java/com/wyl/word/MainActivity.kt
package com.wyl.word

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.findNavController
import androidx.navigation.ui.setupActionBarWithNavController


class MainActivity : AppCompatActivity() {
    private val findNavController by lazy { findNavController(R.id.fragment) }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        setupActionBarWithNavController(findNavController)


    }

    override fun onSupportNavigateUp(): Boolean =
        findNavController.navigateUp() || super.onSupportNavigateUp()
}

